src/generic_ui/polymer/cloud-install.html

```diff
@@ -58,13 +63,39 @@
     <core-signals on-core-signal-open-cloud-install-dialog='{{ openLoginDialog }}'></core-signals>
 
     <!-- TODO: use core-pages rather than multiple dialogs -->
+    <uproxy-dialog layered='false' backdrop='true' id='regionDialog'>
```

Comment:
Ah, yes, I could see that working well too. In other words: minimal number of dialogs.

So, just add a region picker to the `loginDialog` - what do you think, @gitlaura?